python/hsfs/core/feature_group_engine.py

```diff
@@ -221,15 +214,8 @@ def append_features(self, feature_group, new_features):
 
     def update_description(self, feature_group, description):
         """Updates the description of a feature group."""
-        copy_feature_group = fg.FeatureGroup(
-            name=None,
-            version=None,
-            featurestore_id=None,
-            description=description,
-            id=feature_group.id,
-            stream=feature_group.stream,
-            features=feature_group.features,
```

Comment:
check why we needed a copy of the features here

Reply:
it is needed when verifying features after `updateMetadata`
the check looks like this
```
if (featuregroupDTO.getFeatures() != null) {
      cachedFeaturegroupController.verifyPreviousSchemaUnchanged(previousSchema, featuregroupDTO.getFeatures());
      newFeatures = cachedFeaturegroupController.verifyAndGetNewFeatures(previousSchema, featuregroupDTO.getFeatures());
    }
```
An update of the description was tested and everything works as expected.